powerline/bindings/tmux/__init__.py

```diff
@@ -76,6 +76,8 @@ def get_tmux_version(pl):
 	version_string = get_tmux_output(pl, '-V')
 	_, version_string = version_string.split(' ')
 	version_string = version_string.strip()
+	if version_string == 'master':
+		return TmuxVersionInfo(math.inf, 0, version_string)
```

Comment:
Woah, sorry for that. Apparently I accidentally tested this with my 2.3-tagged version twice instead of using the `master` build.

`math.inf` definetely exists in my python 3.6 interpreter, but i changed it to `float('inf')` anyway.